client_packages/cef/login/index.html

```diff
@@ -1,48 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-        <link href="../assets/css/bootstrap.min.css" rel="stylesheet"/>
-        <link href="../assets/css/global-theme.css" rel="stylesheet"/>
-        <link href="style.css" rel="stylesheet"/>
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
I see you've replaced the local files for CDN links, is there a reason why you choose to use external resources? Using external files can add additional loading time onto users which is why CDN's are avoided in this resource.

Also, we already use bootstrap v4.5 and this is v4.

As of now, I wouldn't merge this with CDN links, I'd want these to be local files (see the cef/assets folder)